### DISCUSSION TITLE
(AH) get all scan

### DISCUSSION THREAD

**Discussion by Noobinc1** [chromiecraft/chromiecraft#5481] [Suggestions]
Hi,

would it be possible to implement the get all feature while auction housing?

currently it only returns between 30-50 results I would reduce scans on thus load on the server I think.

No tech background here it is just a suggestion:)

**Comment by 55Honey** [comment]
Hello @Noobinc1,
Blizz has removed this feature in a later expansion for good reason. There are no plans currently to allow / add unlimited AH scanning.

**Reply by Noobinc1** [reply]
> Hello @Noobinc1, Blizz has removed this feature in a later expansion for good reason. There are no plans currently to allow / add unlimited AH scanning.

Hi,

Not sure what the reason is but thanks for taking the time to respond anyway much obliged!

**Reply by heyitsbench** [reply]
> Blizz has removed this feature in a later expansion for good reason.

What the heck would that reason be, if I may ask? Based on a few threads I found, it seems to be something they broke at one point, and the function was split up a few expansions later, but not outright removed. Either way, I'd like to know what that reason would be as it applies to AC/CC.

> There are no plans currently to allow / add unlimited AH scanning.

Might want to close the [respective AC issue](https://github.com/azerothcore/azerothcore-wotlk/issues/5068) then.

**Reply by 55Honey** [reply]
In short: Huge amounts of data being sent for the only purpose of automated buying/selling in real time.
Players are supposed to use the auction house. Not algorithms.
I'm not one to judge what a blizz-like implementation of the AH looks like. But this is beyond the scope of a CC suggestion.
